contracts/WithdrawManager.sol

```diff
@@ -91,6 +92,7 @@ contract WithdrawManager {
         (checkSuccess, callSuccess) = BLS.verifySingle(
             signature,
             pubkey,
+            // TODO What do we do here to verify EIP-712 domain?
             BLS.hashToPoint(appID, abi.encodePacked(msg.sender))
```

Comment:
@ChihChengLiang Your input on how this could/will impact BLS signing and verification is appreciated.

Reply:
The domain is appID. As long as all signing and verification use the same domain it should be fine.
EIP-712 only affects the value of the bytes32 variable.